addons/account_bank_statement_extensions/__init__.py

```diff
@@ -1,22 +1,21 @@
 # -*- encoding: utf-8 -*-
 ##############################################################################
 #
-#    OpenERP, Open Source Management Solution
-#    
-#    Copyright (c) 2011 Noviat nv/sa (www.noviat.be). All rights reserved.
-# 
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
+# Odoo, Open Source Management Solution
+# Copyright (C) 2004-TODAY Odoo S.A. <http://www.odoo.com>
 #
```

Comment:
I'm pretty sure you can't, even if Noviat signed the CLA, they still keep their copyright.